creation/interfaces-and-networks.md

```diff
@@ -692,3 +687,209 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
+`sriov_net`.
+
+It will also render an associated `SriovNetworkNodeState`, which looks like:
+
+```
+apiVersion: v1
+items:
+- apiVersion: sriovnetwork.openshift.io/v1
+  kind: SriovNetworkNodeState
+  metadata:
+    name: sriov-worker
+    namespace: sriov-network-operator
+    ownerReferences:
+    - apiVersion: sriovnetwork.openshift.io/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: SriovNetworkNodePolicy
+      name: default
+  spec:
+    interfaces:
+    - mtu: 1500
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      vfGroups:
+      - deviceType: vfio-pci
+        resourceName: sriov_net
+        vfRange: 0-7
+  status:
+    interfaces:
+    - Vfs:
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.2
+        vendor: 15b3
+        vfID: 0
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.3
+        vendor: 15b3
+        vfID: 1
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.4
+        vendor: 15b3
+        vfID: 2
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.5
+        vendor: 15b3
+        vfID: 3
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.6
+        vendor: 15b3
+        vfID: 4
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.7
+        vendor: 15b3
+        vfID: 5
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:01.0
+        vendor: 15b3
+        vfID: 6
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:01.1
+        vendor: 15b3
+        vfID: 7
+      deviceID: "1015"
+      driver: mlx5_core
+      mtu: 1500
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      totalvfs: 8
+      vendor: 15b3
+    - deviceID: "1015"
+      driver: mlx5_core
+      mtu: 1500
+      name: ens1f1
+      pciAddress: 0000:65:00.1
+      totalvfs: 8
+      vendor: 15b3
+    syncStatus: Succeeded
+```
+
+#### Start an SR-IOV VM
+
+Once all the SR-IOV components are deployed, it is needed to indicate how to
+configure the SR-IOV network. Refer to the following
+`NetworkAttachmentDefinition` for an example:
+
+    apiVersion: "k8s.cni.cncf.io/v1"
+    kind: NetworkAttachmentDefinition
+    metadata:
+      name: sriov-net
+      annotations:
+        k8s.v1.cni.cncf.io/resourceName: openshift.io/sriov_net
+    spec:
+      config: '{
+      "type": "sriov",
+      "ipam": {
+        "type": "host-local",
+        "subnet": "10.56.217.0/24",
+        "rangeStart": "10.56.217.171",
+        "rangeEnd": "10.56.217.181",
+        "routes": [{
+          "dst": "0.0.0.0/0"
+        }],
+        "gateway": "10.56.217.1"
+      }
+    }'
+
+Finally, to create a VM that will attach to the aforementioned Network, refer
+to the following VMI spec:
```

Comment:
Link against the source instead of copying it ? 

https://github.com/redhat-nfvpe/openshift-baremetal-ci/blob/master/tests/kubevirt/vmi/vm-perf.yaml

Reply:
Clicking links requires too much energy (and it may disappear). I prefer to have all the nice examples here.

Reply:
Nothing to do then. For once I got it right.